src/main/java/seedu/commands/AddCommand.java

```diff
@@ -50,6 +51,17 @@ public void execute(ArrayList<String> args) {
                 break;
             }
         }
+        // Error handling: Abort if role or company are missing
+        if (role.isEmpty()) {
```

Comment:
Perhaps can opt to terminate the loop once the invalid role or company is detected? Can try using exceptions